test/ExecutionFramework.cpp

```diff
@@ -57,7 +57,7 @@ ExecutionFramework::ExecutionFramework():
 ExecutionFramework::ExecutionFramework(string const& _ipcPath):
 	m_rpc(RPCSession::instance(_ipcPath)),
 	m_evmVersion(dev::test::Options::get().evmVersion()),
-	m_optimize(dev::test::Options::get().optimize),
+	m_optimiserSettings(dev::test::Options::get().optimize ? solidity::OptimiserSettings::minimal() : solidity::OptimiserSettings::enabled()),
```

Comment:
Not part of this PR, but I find `OptimizerSettings::{minimal, enabled, full}` a bit confusing. And that's shown in this line. I think we could streamline it by changing `enabled` to `default`: `OptimizerSettings::{minimal, default, full}`

Reply:
Hm, but the default is minimal ;)

Reply:
`regular`?

Reply:
OptimizerSettings::{default, regular, full} maybe?

Reply:
-> https://github.com/ethereum/solidity/pull/6334